demo/body3d_two_stage_video_demo.py

```diff
@@ -0,0 +1,338 @@
+import mmcv
+import numpy as np
+
+from mmpose.apis import (get_track_id, inference_pose_lifter_model,
+                         inference_top_down_pose_model, vis_3d_pose_result)
+from mmpose.apis.inference import init_pose_model
+
+try:
+    from mmdet.apis import inference_detector, init_detector
+
+    has_mmdet = True
+except (ImportError, ModuleNotFoundError):
+    has_mmdet = False
+
+
+def process_mmdet_results(mmdet_results, cat_id=1):
+    """Process mmdet results, and return a list of bboxes.
+
+    Args:
+        mmdet_results (list|tuple): mmdet results.
+        cat_id (int): category id (default: 1 for human)
+
+    Returns:
+        person_results (list): a list of detected bounding boxes
+    """
+    if isinstance(mmdet_results, tuple):
+        det_results = mmdet_results[0]
+    else:
+        det_results = mmdet_results
+
+    bboxes = det_results[cat_id - 1]
+
+    person_results = []
+    for bbox in bboxes:
+        person = {}
+        person['bbox'] = bbox
+        person_results.append(person)
+
+    return person_results
+
+
+def dataset_transform(keypoints, pose_det_dataset, pose_lift_dataset):
+    """Transform pose det dataset keypoints convention to pose lifter dataset
+    keypoints convention.
+
+    Args:
+        keypoints (ndarray[K, 2 or 3]): 2D keypoints to be transformed.
+        pose_det_dataset, (str): Name of the dataset for 2D pose detector.
+        pose_lift_dataset (str): Name of the dataset for pose lifter model.
+    """
+    if pose_det_dataset == 'TopDownH36MDataset' and \
+            pose_lift_dataset == 'Body3DH36MDataset':
+        return keypoints
+    elif pose_det_dataset == 'TopDownCocoDataset' and \
+            pose_lift_dataset == 'Body3DH36MDataset':
+        keypoints_new = np.zeros((17, 2))
+        # pelvis is in the middle of l_hip and r_hip
+        keypoints_new[0] = (keypoints[11, :2] + keypoints[12, :2]) / 2
+        # thorax is in the middle of l_shoulder and r_shoulder
+        keypoints_new[8] = (keypoints[5, :2] + keypoints[6, :2]) / 2
+        # head is in the middle of l_eye and r_eye
+        keypoints_new[10] = (keypoints[1, :2] + keypoints[2, :2]) / 2
+        # spine is in the middle of thorax and pelvis
+        keypoints_new[7] = (keypoints_new[0, :2] + keypoints_new[8, :2]) / 2
+        # rearrange other keypoints
+        keypoints_new[[1, 2, 3, 4, 5, 6, 9, 11, 12, 13, 14, 15, 16]] = \
+            keypoints[[12, 14, 16, 11, 13, 15, 0, 5, 7, 9, 6, 8, 10], :2]
+        if keypoints.shape[-1] == 3:
+            keypoints_new = np.concatenate([keypoints_new, keypoints[:, 2:3]],
+                                           axis=1)
+        return keypoints_new
+    else:
+        raise NotImplementedError
+
+
+def main():
+    parser = ArgumentParser()
+    parser.add_argument('det_config', help='Config file for detection')
+    parser.add_argument('det_checkpoint', help='Checkpoint file for detection')
+    parser.add_argument(
+        'pose_detector_config',
```

Comment:
Please use the dash "-" for argument names for consistency. And please also help correct the argument names in 3d_two_stage_img_demo.py.